### ISSUE TITLE
Mosaic Collection Mega Tile

### ISSUE THREAD

**Issue by skleinfeldt** [PSUEducationalEquity/plonetheme.eeq#29]
As a user I can see a collection of top news stories (or other content) on an office landing page, laid out so they look like individual Mosaic Content Tiles (#28). As an editor I can create one of these tiles by choosing an existing Collection of items all of which have lead images. The items in the collection will resize at smaller screen sizes and the images will be dropped at the smallest screen sizes. (See mockup for responsive behavior.)

**Comment by alecpm** [comment]
We should be able to use the tesserae "Collection Summary" tile, which uses the same summary view markup and styling for each item from the collection. We'll probably need to customize the flex box styling within the a bit tile to define the column/row layout of items within the tile.

**Comment by silviot** [comment]
@paulrentschler I merged the work on this and a bunch of other tickets (#9, #10, #11 and #12) to `master`. It would be awesome if you could deploy them (a `git pull` and instance restart should be enough) before today's standup.

**Comment by paulrentschler** [comment]
I created a [collection of news items](https://beta.equity.psu.edu/web/offices/student-disability-resources/news/news) and then added a Collection Summary Tile to the [SDR home page](https://beta.equity.psu.edu/web/offices/student-disability-resources) but there are a few issues:
![image](https://user-images.githubusercontent.com/1778602/149412535-48e5de41-5114-4e61-9736-1a8e7dfe882e.png)

1. Do not like nor want the "View All" link
2. The images are not rendering because they are all referencing the collection URL for the image and also linking to the collection URL rather than the news item URL
3. The title also links to the collection URL rather than the news item URL
4. The tile is not rendering in a 2-column grid at wider widths

**Comment by silviot** [comment]
@paulrentschler I fixed the issues mentioned, except for `1.` that can be solved by unticking `Show collection title` in the tile edit dialog:
![image](https://user-images.githubusercontent.com/396425/149506950-8838f826-3738-4aa2-827a-820ae6b31462.png)
I changed the linked office already.

**Comment by paulrentschler** [comment]
@silviot Unticking `Show collection title` does get rid of the `View All` link but it also gets rid of the `News` title which we would like to keep. If hiding the `View All` with a CSS rule is an easy fix, I am okay with that.

**Comment by silviot** [comment]
@paulrentschler Right! I didn't think about the title which would go missing.
[I added a CSS rule](https://github.com/PSUEducationalEquity/plonetheme.eeq/commit/4845411ea6c8377b798f9bb7463d8b90ecb67a73) to hide the `View all` link.

**Comment by paulrentschler** [comment]
@silviot no worries. I deployed the changes prior to that CSS rule and I am still not seeing the images and the titles still link to the news collection rather than the individual news items.

Titles should be left justified, not centered and we need some padding between the tiles.

**Comment by silviot** [comment]
@paulrentschler These issues shoudl  be fixed now in `master`.

**Comment by paulrentschler** [comment]
The title and summary text is `justified` and we want it to be `left justified`.

**Comment by silviot** [comment]
Right! I realized I've been interpreting the word _justified_ wrong my whole life. [I pushed a fix](https://github.com/PSUEducationalEquity/plonetheme.eeq/commit/4af1775738703650fd8381d8f755a717e25e587a).

**Comment by paulrentschler** [comment]
The left justified text looks good now. Upon further testing, we noticed that the two columns do not collapse down into a single column on small devices. I suspect this might be related to the same issue with #28

**Comment by alecpm** [comment]
I'm seeing the two columns collapse now, even though the fix for 28 is not yet deployed. Was the equal height tile format applied and then removed on this?

**Comment by paulrentschler** [comment]
@alecpm Mike and I am not sure if the equal height tile format was applied and then removed; the tiles are equal height now. The right side portlet column is collapsing correctly but the two news items side by side are not collapsing one underneath the other on smaller devices.
![image](https://user-images.githubusercontent.com/1778602/153617241-e1e1ca01-c183-493c-84b4-1eba9518ca51.png)

**Comment by alecpm** [comment]
@paulrentschler this issue isn't related to the Equal Height styling, but is explicit in the CSS. The collection summary tile always displays tiles 2 across independent of device size or row format. It's a pretty simple fix to make them stack on mobile if you'd prefer that.

I think that independently added summary tiles (i.e. not generated from a collection) this should work as expected, the collection summary tile styling has a hardcoded internal column structure which doesn't collapse.

**Comment by MBrahosky** [comment]
@alecpm If at all possible Paul and I would like to see this changed so that the collection summary tile collapses.

**Comment by alecpm** [comment]
@MBrahosky @paulrentschler I've pushed a CSS change that should make the collection summary tile collapse as expected. You'll need to pull in the changes and run buildout as usual.

**Comment by paulrentschler** [comment]
@alecpm the fix looks good. Thanks.

Mike is on vacation through next week so I am going to leave this here till he gets back and can also review it.
